app/views/shared/_dap_analytics.html.erb

```diff
@@ -1,4 +1,3 @@
 <!-- <%= t('notices.dap_participation') %> -->
 <% dap_source = 'https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=GSA&subagency=TTS' %>
-<%= backwards_compatible_javascript_tag({ src: dap_source, async: true, id: '_fed_an_ua_tag' }) do %>
-<% end %>
+<%= javascript_tag(src: dap_source, async: true, id: '_fed_an_ua_tag', nonce: true) %>
```

Comment:
I wonder if this even needs `nonce` 🤔

Reply:
doesn't our CSP require a nonce for all JS?

https://github.com/18F/identity-idp/blob/2ff7db662327125ec3e769e41a896fa1819ce23e/config/initializers/content_security_policy.rb#L62-L65

Reply:
> I wonder if this even needs `nonce` 🤔

I just checked source of https://secure.login.gov and it doesn't assign the `nonce` attribute. I'm assuming Rails bails when it sees there's no content to be nonced.

Reply:
> doesn't our CSP require a nonce for all JS?

For scripts loaded over the network, the CSP constraint is satisfied by the `'self'` directive. It's only for inline JavaScript that we need these nonce attributes.

Reply:
oh hm ok

Also wow I think putting the session ID (the cookie value) in the HTML source like that is not good, I wish we could hash it or pick something different

Reply:
Hm, yeah, I'd also expect it to be unique per instance too. @jmhooper do you recall if there was a particular rationale here? In other [examples](https://github.com/rails/rails/blob/3872bc0e54d32e8bf3a6299b0bfe173d94b072fc/actionview/test/ujs/server.rb#L37) I see that it's configured to generate a random value?

Reply:
Since it's not related, I'm gonna merge as-is. I was planning to make a ticket to track that, though I'd want to confirm there's actually something to be done.